### PR TITLE
fix: OAuth controls

### PR DESCRIPTION
**Pull request by imaretic**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Updates the UI for controlling the Spreadsheet-Gateway and its credentials.

* **What is the current behavior?** (You can also link to an open issue here)
Currently, it works, but the CSS needs some updating, to improve UX.

* **What is the new behavior (if this is a feature change)?**
With this change, there are now two options: to reset app credentials or to reset user credentials, depending on what the user wants to perform. The overall style is also updated so it's all aligned well.

* **Other information**:

**Status**

- [ ] Code documentation for the relevant parts in the code has been added/updated by the PR author
- [x] The functionality has been tested by the PR author
- [ ] The functionality has been tested by NRK

### CODE REVIEW

File: meteor/client/ui/Settings/components/ConfigManifestOAuthFlow.tsx [hunk 1]
```diff
@@ -105,15 +135,23 @@ export const ConfigManifestOAuthFlowComponent = withTranslation()(
 						// If this is set, we have completed the authentication procedure.
 						// A reset button is provided to begin the flow again if authorization is revoked by the user.
 						<div className="mod mvs mhs">
-							<button className="btn btn-secondary" onClick={() => this.resetAuthentication()}>
-								{t('Reset Authentication')}
+							<button
+								className="btn btn-secondary btn-tight"
+								onClick={() => this.resetAppCredentials()}
+								style={{ marginRight: '10px' }}
```
Comment (jstarpl) [inline meteor/client/ui/Settings/components/ConfigManifestOAuthFlow.tsx:141]: Could this be changed to a utility class instead? I think `mrs` utility class has the same effect.

File: meteor/client/ui/Settings/components/ConfigManifestOAuthFlow.tsx [hunk 2]
```diff
@@ -134,18 +172,22 @@ export const ConfigManifestOAuthFlowComponent = withTranslation()(
...
-									<div className="mdi">
-										<div>{t('Click on the link below and accept the permissions request.')}</div>
-										<div>
+									<div>
+										<div>{t('Click on the button below and accept the permissions request.')}</div>
+										<div className="mvs">
 											{device.accessTokenUrl ? (
 												<a href={device.accessTokenUrl} target="_blank" rel="noreferrer">
-													{device.accessTokenUrl}
+													<button className="btn btn-primary">{t('Authorize App Access')}</button>
```
Comment (jstarpl) [inline meteor/client/ui/Settings/components/ConfigManifestOAuthFlow.tsx:180]: A `button` in an `<a href>` doesn't seem right. I think this should work OK with just having an `<a className='btn'>`. If not - `window.open`?

**Review by jstarpl — CHANGES_REQUESTED** [review]

**Comment by imaretic** [comment]
In the new commit:
- CSS styling has been replaced with the utility class
- Button is not inside `<a>` anymore - of course
- Resetting app credentials won't kill the spreadsheet gateway anymore
- Resetting user credentials will only delete user credentials and not app credentials

**Comment by imaretic** [comment]
In the new commit:

- CSS styling has been replaced with the utility class
- Button is not inside `<a>` anymore - of course
- Resetting app credentials won't kill the spreadsheet gateway anymore
- Resetting user credentials will only delete user credentials and not app credentials

**Review by jstarpl — APPROVED** [review]

**Pull request merged**